src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -246,7 +271,14 @@ def _aws_s3_policy(self, s3_prefixes, account_id, role_name, kms_key=None):
         s3_prefixes_enriched = sorted([self.S3_PREFIX + s3_prefix for s3_prefix in s3_prefixes_strip])
         statement = [
             {
-                "Action": ["s3:GetObject", "s3:PutObject", "s3:DeleteObject", "s3:ListBucket", "s3:GetBucketLocation"],
+                "Action": [
+                    "s3:GetObject",
+                    "s3:PutObject",
+                    "s3:PutObjectAcl",
```

Comment:
our doc does not require this permission - https://docs.databricks.com/en/connect/unity-catalog/storage-credentials.html#step-1-create-an-iam-role
```suggestion
```

Reply:
It is required.